editor/gui/editor_file_dialog.cpp

```diff
@@ -1110,6 +1120,21 @@ void EditorFileDialog::_filter_selected(int) {
 	update_file_list();
 }
 
+void EditorFileDialog::_filename_filter_changed() {
+	update_filename_filter();
+	item_list->call_deferred("select", 0);
+	update_file_list();
```

Comment:
```suggestion
	update_file_list();
	if (item_list->get_item_count() > 0) {
		item_list->select(0);
	}
```

Same here, `update_file_list` can result in the empty list and "select" will print error.